### ISSUE TITLE
how's the auto-complete setting?

### ISSUE THREAD

**Issue by Vbubblery** [TeamWertarbyte/material-ui-chip-input#249]
In the Properties, there are dataSource and dataSourceConfig could use for autocomplete. But no example can explain how to implement. And I checked example, there is an only react-autocomplete example, so that means these two properties are not working now?

**Comment by leMaik** [comment]
Exactly. There is no and there will be no built-in autocomplete. Implementing auto-completion is somewhat out of scope. The react-autocomplete example shows how to integrate a autocomplete library, so if you want to use a different one, it should also be possible. Let me know if you need any help with that. I'd also be happy to add an example for integrating another library. :+1:

Developing auto-completion is not trivial, so we should outsource that into different libraries that do auto-completion only, and do it well.

**Comment by admehta01** [comment]
Should update documentation for `dataSource` and `dataSourceConfig` so people don't try it out autocomplete. It still says they're used for autocomplete.
